ERC20_LIST.json

```diff
@@ -292,69 +292,83 @@
-        "symbol": "ANGLE",
         "hasPermit": false,
-        "logoURI": "https://raw.githubusercontent.com/AngleProtocol/angle-assets/main/0_tokens/ANGLE-token/ANGLE%20Token-light.svg"
+        "logoURI": "https://raw.githubusercontent.com/AngleProtocol/angle-assets/main/0_tokens/ANGLE-token/ANGLE%20Token-light.svg",
+        "name": "Angle Governance Token",
+        "symbol": "ANGLE"
       },
       "0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619": {
-        "name": "Wrapped Ether",
         "decimals": 18,
-        "symbol": "WETH",
         "hasPermit": false,
-        "useInSwap": true,
-        "logoURI": "https://raw.githubusercontent.com/AngleProtocol/angle-token-list/main/src/assets/tokens/WETH.svg"
+        "logoURI": "https://raw.githubusercontent.com/AngleProtocol/angle-token-list/main/src/assets/tokens/WETH.svg",
+        "name": "Wrapped Ether",
+        "symbol": "WETH",
+        "useInSwap": true
       },
       "0x0000000000000000000000000000000000001010": {
-        "name": "Matic Token",
         "decimals": 18,
-        "symbol": "MATIC",
         "hasPermit": false,
-        "useInSwap": true,
-        "logoURI": "https://raw.githubusercontent.com/AngleProtocol/angle-token-list/main/src/assets/tokens/MATIC.svg"
+        "logoURI": "https://raw.githubusercontent.com/AngleProtocol/angle-token-list/main/src/assets/tokens/MATIC.svg",
+        "name": "Matic Token",
+        "symbol": "MATIC",
+        "useInSwap": true
       },
       "0x1BFD67037B42Cf73acF2047067bd4F2C47D9BfD6": {
-        "name": "Wrapped Bitcoin",
         "decimals": 8,
-        "symbol": "WBTC",
-        "useInSwap": true,
         "hasPermit": false,
-        "logoURI": "https://raw.githubusercontent.com/AngleProtocol/angle-token-list/main/src/assets/tokens/WBTC.svg"
+        "logoURI": "https://raw.githubusercontent.com/AngleProtocol/angle-token-list/main/src/assets/tokens/WBTC.svg",
+        "name": "Wrapped Bitcoin",
+        "symbol": "WBTC",
+        "useInSwap": true
       },
       "0x0d500B1d8E8eF31E21C99d1Db9A6444d3ADf1270": {
-        "name": "Wrapped Matic",
         "decimals": 18,
+        "hasPermit": false,
+        "logoURI": "https://polygonscan.com/token/images/wMatic_32.png",
+        "name": "Wrapped Matic",
         "symbol": "WMATIC",
+        "useInSwap": true
+      },
+      "0xE7a24EF0C5e95Ffb0f6684b813A78F2a3AD7D171": {
+        "decimals": 18,
         "hasPermit": false,
-        "useInSwap": true,
-        "logoURI": "https://polygonscan.com/token/images/wMatic_32.png"
+        "isLP": true,
+        "logoURI": "https://raw.githubusercontent.com/AngleProtocol/angle-token-list/main/src/assets/tokens/USDC.svg",
+        "name": "Curve.fi amDAI/amUSDC/amUSDT",
+        "symbol": "am3CRV",
+        "underlyingTokens": [
+          "0x27F8D03b3a2196956ED754baDc28D73be8830A6e",
+          "0x1a13F4Ca1d028320A707D99520AbFefca3998b7F",
+          "0x60D55F02A771d515e077c9C2403a1ef324885CeC"
+        ],
+        "useInSwap": true
```

Comment:
so what does it mean `useInSwap`?

Reply:
for swap token list, but we can get rid of it